### DISCUSSION TITLE
Background

### DISCUSSION THREAD

**Discussion by EdguyDeMaupassant** [martinet101/ElevenClock#660] [Q&A] [answered]
Hey, I would like to make the background of the clock transparent, but if I make this we can see the original clock under it, how can I fix that ?

**Comment by marticliment** [comment, marked as answer]
This can't be really made, the background color is to hide the original clock.

**Reply by EdguyDeMaupassant** [reply]
so we can't, for exemple, just desactivate the new clock from the main monitor ?

**Reply by marticliment** [reply]
Yes, you can do that also from the settings window
